jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory.md

```diff
@@ -3014,6 +3015,54 @@ The first one uses [Using Placeholders](https://docs.jfrog-applications.jfrog.io
 }
 ```
 
+**Example 15:**
+
+Create release bundle with all files retrieved by the AQL query and set their target after distirbution to the **froggy** directory at the edges.
```

Comment:
* Is the target the "froggy directory" or "froggy repository"?
* Since this example appears to be the only area in the documentation introducing the mappings functionality, we should elaborate a lot more here, to ensure the users fully understand what this functionality is for and how to use it. I recommend using an AI + grammar correction tool to create this description.

Reply:
Got it makes total sense